### PR TITLE
Update to the Angular 9

### PR DESCRIPTION
**Pull request by ye3i**
Update to the Angular 9

### CODE REVIEW

File: projects/ngx-permissions/tsconfig.lib.prod.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```
Comment (AlexKhymenko) [inline projects/ngx-permissions/tsconfig.lib.prod.json]: Hi. Why Ivy is disabled?

**Comment by AlexKhymenko** [comment]
Thank You for pull request.!!!! Really appreciate this just don't understand disabling ivy?

**Comment by hugoj-goncalves** [comment]
You can read more about it here

https://angular.io/guide/ivy#maintaining-library-compatibility

**Comment by AlexKhymenko** [comment]
Again Thank You very much!!!

File: projects/ngx-permissions/tsconfig.lib.prod.json (hunk 1 above)
Comment (ye3i) [reply projects/ngx-permissions/tsconfig.lib.prod.json]: Is not yet recommended by the Angular team.
According to [this table](https://gist.github.com/LayZeeDK/61caba93df1ec1a0788c94a973c8dfac#file-angular-view-engine-to-ivy-transition-csv) Ivy should be enabled with Angular 10

**Pull request merged**